src/OpenCensus.Exporter.Stackdriver/Implementation/Constants.cs

```diff
@@ -0,0 +1,39 @@
+﻿// <copyright file="ApplicationInsightsExporter.cs" company="OpenCensus Authors">
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of theLicense at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenCensus.Exporter.Stackdriver.Implementation
+{
+    using System;
+
+    internal class Constants
+    {
+        public const string LABEL_DESCRIPTION = "OpenCensus TagKey";
+        public const string OPENCENSUS_TASK = "opencensus_task";
+        public const string OPENCENSUS_TASK_DESCRIPTION = "Opencensus task identifier";
+        public const string GCP_GKE_CONTAINER = "k8s_container";
+        public const string GCP_GCE_INSTANCE = "gce_instance";
+        public const string AWS_EC2_INSTANCE = "aws_ec2_instance";
```

Comment:
I don't see you using those. Is it copy/paste from somewhere?

Reply:
This is from java exporter. 
Stackdriver names (prefix) metrics based on the environment in which they are collected. I am going to use these later on in the implementation. (future PRs).